packages/stylelint-config/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
Is this license file necessary? It looks like a standard one. I have no idea myself, just asking. We don't put copies for "GPL-2.0-or-later" in other packages.

Reply:
Good question, I'm not sure either. Wondering who would be best to answer this?

At least [the license itself says](https://github.com/WordPress-Coding-Standards/stylelint-config-wordpress/blob/7816e565b44cdb16969297f967343f6c8b092be6/LICENSE#L12-L13), that it should be included. But the [copyright info](https://github.com/WordPress-Coding-Standards/stylelint-config-wordpress/blob/7816e565b44cdb16969297f967343f6c8b092be6/LICENSE#L3) is actually a bit odd. 🤷🏻‍♂️